plugin/storage/cassandra/spanstore/operation_names.go

```diff
@@ -25,23 +26,27 @@ import (
 	"github.com/jaegertracing/jaeger/pkg/cache"
 	"github.com/jaegertracing/jaeger/pkg/cassandra"
 	casMetrics "github.com/jaegertracing/jaeger/pkg/cassandra/metrics"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
 const (
-	insertOperationName = `INSERT INTO operation_names(service_name, operation_name) VALUES (?, ?)`
-	queryOperationNames = `SELECT operation_name FROM operation_names WHERE service_name = ?`
+	operationTableName        = `operations`
+	insertOperationName       = `INSERT INTO ` + operationTableName + ` (service_name, span_kind, operation_name) VALUES (?, ?, ?)`
+	queryOperationNames       = `SELECT span_kind, operation_name FROM ` + operationTableName + ` WHERE service_name = ? `
+	queryOperationNamesByKind = `SELECT span_kind, operation_name FROM ` + operationTableName + ` WHERE service_name = ? AND span_kind = ? `
```

Comment:
We need to be backwards compatible with previous DB schema. On startup we can [interrogate table metadata][1] to check if the table has the extra column or not.

[1]: https://stackoverflow.com/questions/38696316/how-to-list-all-cassandra-tables/50163285

Reply:
We have backward compatibility on the api (endpoint) level, which returns []string as operations for the old API, the old UI will work when we deploy new jaeger-query (verified in staging).
Given only UI is the user of the api, i think it's good enough.

Do we have any specific reason that we need to be backward compatible on DB schema level?